Systems/c172p-damage.xml

```diff
@@ -243,38 +243,39 @@
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-high
             </test>
+            
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-medium
                 damage/roll-moment GT 4000.0
-                wing-damage/right-wing GT 0.0
+                wing-damage/right-wing LT 1.0
```

Comment:
I see no reason to set a condition `wing-damage/right-wing GT 0.0` to the other wing. We can break the left one even if the right one is OK.

Farther...
Initially, I added `wing-damage/right-wing LT 1.0` to break the left wing. This was in order to avoid breaking the right wing as soon as the left one was broken (by a roll moment). But it seems useless, the right wing holds as long as the high velocity is not reached. I think that this condition can be deleted too.